new-site/src/docs/elements/components/navigation/code.mdx

```diff
@@ -0,0 +1,194 @@
+    <Navigation.Item label="Item" />
+    <Navigation.Item label="Item" />
+  </Navigation.Row>
+</Navigation>
+```
+
+</Playground>
+
+#### Small (1 row)
+<Playground>
+
+
+```jsx
+<Navigation
+  title="Helsinki Design System"
+  menuToggleAriaLabel="menu"
+  skipTo="#content"
+  skipToContentLabel="Skip to content"
+>
+  <Navigation.Row variant="inline">
+    <Navigation.Item label="Item" active />
+    <Navigation.Item label="Item" />
+    <Navigation.Item label="Item" />
+  </Navigation.Row>
+</Navigation>
+```
+
+</Playground>
+
+#### Navigation actions
+<Playground>
+
+
+```jsx
+<Navigation
+  title="Helsinki Design System"
+  menuToggleAriaLabel="menu"
+  skipTo="#content"
+  skipToContentLabel="Skip to content"
+>
+  <Navigation.Actions>
+    <Navigation.Search searchLabel="Search" searchPlaceholder="Search page" />
+    <Navigation.User label="Sign in" />
+    <Navigation.LanguageSelector label="FI">
+      <Navigation.Item lang="fi" label="Suomeksi" />
+      <Navigation.Item lang="sv" label="På svenska" />
+      <Navigation.Item lang="en" label="In English" />
+    </Navigation.LanguageSelector>
+  </Navigation.Actions>
+</Navigation>
+```
+
+</Playground>
+
+#### Colour variations
+<Playground>
+
+
+```jsx
+<>
+<Navigation
+  title="Helsinki Design System"
+  menuToggleAriaLabel="menu"
+  skipTo="#content"
+  skipToContentLabel="Skip to content"
+>
+  <Navigation.Row variant="inline">
+    <Navigation.Item label="Item" active />
+    <Navigation.Item label="Item" />
+    <Navigation.Item label="Item" />
+  </Navigation.Row>
+</Navigation>
+<br />
+<Navigation
+  theme="dark"
+  title="Helsinki Design System"
+  menuToggleAriaLabel="menu"
+  skipTo="#content"
+  skipToContentLabel="Skip to content"
+>
+  <Navigation.Row variant="inline">
+    <Navigation.Item label="Item" active />
+    <Navigation.Item label="Item" />
+    <Navigation.Item label="Item" />
+  </Navigation.Row>
+</Navigation>
+</>
+```
+
+</Playground>
+
+
+### Packages
+| Package           | Included                      | Storybook link                                                                                                                      | Source link   |
+| ----------------- | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| **HDS React**     | <IconCheckCircleFill /> Yes   | <Link size="M" external href="/storybook/react/?path=/story/components-navigation--example">View in Storybook</Link> | <Link size="M" external href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/react/src/components/navigation">View source</Link> |
+| **HDS Core**      | <IconCrossCircle /> No        | - | - |
+
+### Properties
+Note! You can find the full list of properties in the <Link size="M" href="/storybook/react/?path=/story/components-logo" external>React Storybook.</Link>
```

Comment:
The link is broken. Also, at least at the moment, React Storybook does not list all properties.

Reply:
All Storybook links will be broken as long as the Storybook is not hosted under the same domain.